cudamapper/src/overlapper.cpp

```diff
@@ -16,22 +16,6 @@ namespace claragenomics
 namespace cudamapper
 {
 
-void Overlapper::filter_overlaps(std::vector<Overlap>& filtered_overlaps, const std::vector<Overlap>& overlaps,
```

Comment:
@tijyojwad @vellamike
Should this function be in `overlapper.cpp` or `overlapper_triggered.cu`? I have a feeling it is not `overlapper_triggerd` specific

Reply:
@mimaric agreed, it's not overlapper_triggered specific. I mostly followed the example of the FusedOp (since that didn't look overlapper_triggered specific either). Do you think it makes sense to move both of them to a common overlapper_ops.hpp file? (don't think it should really belong to the public overlapped.hpp either)

Reply:
Ok, let's keep it as it is. Overlapper fill need a separate refactoring anyway